org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend

```diff
@@ -44,5 +44,177 @@ class NumberTestCase extends AbstractWollokInterpreterTestCase {
 		}
 		'''.interpretPropagatingErrors
 	}
+
+	@Test
+	def void integersBetwenTrue() {
+		'''program a {
+		
+			assert.that(3.between(1, 5))
+		
+		}
+		'''.interpretPropagatingErrors
```

Comment:
I was thinking, maybe we can provide a method similar to "interpretPropagatingErrors" to avoid the bolierplate of the "program" and just write the assertion for this cases, right ?
Something like

```xtend
def void integersBetweenTrue() {
    '''assert.that(3.between(1,5)'''.test
}
```

Test would do
```xtend
def test(String testCode) {
    ''''program a {
           <<testCode>>
    '''.interpretPropagatingErrors
}
```

Just a toy (un chiche :) )